scripts/src/primazactl/primazaworker/parser.py

```diff
@@ -0,0 +1,142 @@
+import os
+import argparse
+from pathlib import Path
+from primazactl.errors import AtLeastOneError
+from primazactl.types import \
+    existing_file, kubernetes_name, semvertag_or_latest
+from primazactl.primazamain.primazamain import PrimazaMain
+from .primazaworker import PrimazaWorker
+from primazactl.utils.kubeconfig import from_env
+
+
+def add_group(subparsers, parents=[]):
+    worker_group = subparsers.add_parser(
+        name="worker",
+        help="Operations on main Worker cluster",
+        parents=parents)
+    worker_subparsers = worker_group.add_subparsers()
+
+    add_subparser_join(worker_subparsers, parents=parents)
+
+
+def add_subparser_join(parser: argparse.ArgumentParser, parents=[]):
+    install_parser = parser.add_parser(
+        name="join",
+        help="Join Primaza",
+        parents=parents)
+    install_parser.set_defaults(func=join_primaza)
+    add_args_join(install_parser)
```

Comment:
Not sure I like the use of "join" partly because I am not sure what we pick as the reverse "leave"?? What is good join and install need different flags and this makes it easy.

Reply:
I'd say that we will eventually provide a `leave` command.

Reply:
OK, if you are happy with "leave" all good by me. Thinking of create namespace, could that be considered as a join too?